osu.Game.Rulesets.Diva/Difficulty/DivaDifficultyCalculator.cs

```diff
@@ -18,27 +18,24 @@ public DivaDifficultyCalculator(Ruleset ruleset, WorkingBeatmap beatmap)
         {
         }
 
-        protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beatmap, Mod[] mods, Skill[] skills, double clockRate) {
+        protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beatmap, Mod[] mods, Skill[] skills, double clockRate)
+        {
```

Comment:
Moving the brace on its own line is personal preference, check with @Artemis-chan if they're okay with that

Reply:
it's on a new line everywhere else

Reply:
Ah, figures.